.gitignore

```diff
@@ -5,3 +5,4 @@
 inst/doc
 redcap_select_rename_code.txt
 redcap_select_rename_overview.csv
+data-raw
```

Comment:
this seems to conflict with your commits - you've commited the data-raw folder...? Is it needed here?

Reply:
yes, this folder is not really necessary on the github page as it contains the preparation of the demodata. can we just delete it?

Reply:
it is useful on github in case we need to modify/recreate the dataset at some point. my question was whether it needs to be in the ignore list...